Makefile

```diff
@@ -89,15 +89,6 @@ $(BUILTIN_ACTORS_BUNDLE): $(BUILTIN_ACTORS_CODE)
 	rustup target add wasm32-unknown-unknown && \
 	cargo run --release -- -o output/$(shell basename $@)
 
-
-# Compile the ABI artifacts and Rust bindings for the IPC Solidity actors.
-ipc-actors-abi: $(IPC_ACTORS_ABI)
-	cargo build --release -p fendermint_vm_ipc_actors
-
-# Force reompilation of the IPC actors.
-clean-ipc-actors-abi:
```

Comment:
I'd also leave this target while we need the ABI files for docker, as otherwise I'd have to go into the other repo and do a `git pull` to refresh it, because otherwise it doesn't do it.

It would be nice if we could find a way to link to the source that cargo downloaded due to the project reference and get the stuff from there, so there's no chance for a mismatch.